src/components/spinner/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './style.module.scss';
```

Comment:
Enter empty string between this imports and replace imports order in other files like: main react libraries - custom react libraries - your components - styles